docs/chat_message_history.ipynb

```diff
@@ -152,31 +134,62 @@
     "TABLE_NAME = \"message_store\"  # @param {type: \"string\"}\n",
     "USER = \"my-user\"  # @param {type: \"string\"}\n",
     "PASSWORD = input(\"Please provide a password to be used for the database user: \")"
-   ]
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "\n",
+    "If you are using El Carro, you can find the hostname and port values in the\n",
+    "status of the El Carro Kubernetes instance.\n",
+    "Use the user password you created for your PDB.\n",
+    "Example"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "kubectl get -w instances.oracle.db.anthosapis.com -n db\n",
+    "NAME   DB ENGINE   VERSION   EDITION      ENDPOINT      URL                DB NAMES   BACKUP ID   READYSTATUS   READYREASON        DBREADYSTATUS   DBREADYREASON\n",
+    "mydb   Oracle      18c       Express      mydb-svc.db   34.71.69.25:6021                          False         CreateInProgress"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "### ElCarroEngine Connection Pool\n",
     "\n",
     "`ElCarroEngine` configures a connection pool to your Oracle database, enabling successful connections from your application and following industry best practices."
-   ]
+   ],
+   "metadata": {
+    "collapsed": false
+   }
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "id": "xG1mYFkEWbkp"
+   },
    "outputs": [],
    "source": [
     "from langchain_google_el_carro import ElCarroEngine\n",
     "\n",
     "elcarro_engine = ElCarroEngine.from_instance(\n",
-    "    db_host=HOST,\n",
-    "    db_port=PORT,\n",
-    "    db_name=DATABASE,\n",
-    "    db_user=USER,\n",
-    "    db_password=PASSWORD,\n",
+    "    db_host=\"127.0.0.1\",\n",
```

Comment:
Why not the parameters defined above?

Reply:
want to move it around so that if they are connecting to oracle from another source, and then mention below what they should do if they are using el carro.